src/main/kotlin/com/openlattice/search/SearchService.kt

```diff
@@ -493,201 +495,246 @@ class SearchService(
         )
     }
 
```

Comment:
This diff is a bit of a mystery to me. Is there a better commit to review this one on than the final branch?